src/screens/delegation-screen/selected-baker-screen/baker-rewards-list/baker-rewards-list.tsx

```diff
@@ -22,15 +24,17 @@ const allRewardsPerEventKeys: (keyof RewardsPerEventHistoryItem)[] = [
   'rewardPerFutureEndorsement'
 ];
 
+const AVERAGE_ITEM_HEIGHT = 280;
```

Comment:
Did you mean `Math.floor(formatSize(280))`?

Reply:
look at the 1 comment thread